src/main/java/frc/team4909/robot/Robot.java

```diff
@@ -89,7 +93,7 @@ public void autonomousPeriodic() {
    */
   @Override
   public void teleopPeriodic() {
-
+    System.out.println(lidar.getDistance());
```

Comment:
why is this being printed out? that's going to clutter our logs– at least add some descriptive data here

Reply:
Ok.

Reply:
Nah, testing is still necessary - I'd like to keep this printing for now - I will however add a comment that this can be removed for competition.